src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -156,15 +155,9 @@ const Title = styled.h1`
   font-weight: inherit;
 `
 
-// TODO: min-max font size responsive to vp
-// font-size: calc([minimum size] + ([maximum size] - [minimum size]) * ((100vw - [minimum viewport width]) / ([maximum viewport width] - [minimum viewport width])))
 const TitleBlock = styled.span<{ textAlign?: string }>`
   display: block;
-  font-size: 20vh;
+  font-size: calc(80px + (200 - 80) * ((100vw - 300px) / (1600 - 300)));
```

Comment:
Minor, but it might be helpful to include the previous comments here as well for future readers. Otherwise these numbers feel a bit arbitrary.

```
font-size: calc([minimum size] + ([maximum size] - [minimum size]) * ((100vw - [minimum viewport width]) / ([maximum viewport width] - [minimum viewport width])))
```